module-3/assignments/activity-1.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<emp:employeeList xmlns:emp="http://ABC.com/employeeList">
+<List:personList>
+    <EMP:empID>E0000001</EMP:empID>
+    <DEP:name>Sales</DEP:name>
```

Comment:
```suggestion
<list:employeeList xmlns:emp="http://ABC.com/employeeList" xmlns:list="" xmlns:dep="">
<list:personList>
    <emp:empID>E0000001</emp:empID>
    <dep:name>Sales</dep:name>
```

you need to import `list` and `dep` namespaces as well
employeeList is part of `list` namespace
prefer using lower case in xml